storefront/modules/products/components/WebPayment.js

```diff
@@ -35,7 +36,9 @@ const WebPayment = ({ onClick, onSuccess = null }) => {
         </button>
       );
     }
-  } catch (e) {}
+  } catch (e) {
+    console.error(e);
```

Comment:
Better, but I'd rather remove this catch block completely. It is better to fail loudly than to swallow errors silently.